src/tests/testScenarios.json

```diff
@@ -0,0 +1,169 @@
+      ]
+    },
+    "wstETH15x": {
+      "setAmounts": ["1", "10"],
+      "inputTokens": [
+        { "symbol": "USDC", "exchangeRate": 5000 },
+        { "symbol": "ETH", "exchangeRate": 2 },
+        { "symbol": "WETH", "exchangeRate": 2 }
+      ]
+    }
+  },
+  "1": {
+    "ETH2X": {
+      "setAmounts": ["1", "10", "100"],
+      "inputTokens": [
+        { "symbol": "USDC", "exchangeRate": 300 },
+        { "symbol": "ETH", "exchangeRate": 0.1 },
+        { "symbol": "WETH", "exchangeRate": 0.1 }
+      ]
+    },
+    "BTC2X": {
+      "setAmounts": ["1", "10", "100"],
+      "inputTokens": [
+        { "symbol": "USDC", "exchangeRate": 3000 },
+        { "symbol": "ETH", "exchangeRate": 1 },
+        { "symbol": "WETH", "exchangeRate": 1 }
+      ]
+    },
+    "hyETH": {
+      "setAmounts": ["1", "10", "100"],
+      "inputTokens": [
+        { "symbol": "ETH", "exchangeRate": 2 },
+        { "symbol": "WETH", "exchangeRate": 2 }
+      ]
+    }
+  },
+  "42161": {
+    "ETH2X": {
+      "setAmounts": ["1", "10", "100"],
+      "inputTokens": [
+        { "symbol": "USDC", "exchangeRate": 300 },
+        { "symbol": "ETH", "exchangeRate": 0.1 },
+        { "symbol": "WETH", "exchangeRate": 0.1 }
+      ]
+    },
+    "BTC2X": {
+      "setAmounts": ["1", "10", "20"],
+      "inputTokens": [
+        { "symbol": "USDC", "exchangeRate": 3000 },
+        { "symbol": "ETH", "exchangeRate": 1 },
+        { "symbol": "WETH", "exchangeRate": 1 }
+      ]
+    },
+    "ETH3X": {
+      "setAmounts": ["1", "10", "100"],
+      "inputTokens": [
+        { "symbol": "USDC", "exchangeRate": 300 },
+        { "symbol": "ETH", "exchangeRate": 0.1 },
+        { "symbol": "WETH", "exchangeRate": 0.1 }
+      ]
+    },
+    "BTC3X": {
+      "setAmounts": ["1", "10", "20"],
+      "inputTokens": [
+        { "symbol": "USDC", "exchangeRate": 3000 },
+        { "symbol": "ETH", "exchangeRate": 1 },
+        { "symbol": "WETH", "exchangeRate": 1 }
+      ]
+    },
+    "iETH1X": {
+      "setAmounts": ["1", "10", "20"],
+      "inputTokens": [
+        { "symbol": "USDC", "exchangeRate": 3000 },
+        { "symbol": "ETH", "exchangeRate": 2 },
+        { "symbol": "WETH", "exchangeRate": 2 }
+      ]
+    },
+    "iBTC1X": {
+      "setAmounts": ["1", "10", "20"],
+      "inputTokens": [
+        { "symbol": "USDC", "exchangeRate": 3000 },
+        { "symbol": "ETH", "exchangeRate": 2 },
+        { "symbol": "WETH", "exchangeRate": 2 }
+      ]
+    },
+    "BTC2xETH": {
+      "setAmounts": ["1", "10", "20"],
+      "inputTokens": [
+        { "symbol": "USDC", "exchangeRate": 3000 },
+        { "symbol": "ETH", "exchangeRate": 2 },
+        { "symbol": "WETH", "exchangeRate": 2 }
+      ]
+    },
+    "ETH2xBTC": {
+      "setAmounts": ["1", "10", "20"],
+      "inputTokens": [
+        { "symbol": "USDC", "exchangeRate": 3000 },
+        { "symbol": "ETH", "exchangeRate": 2 },
+        { "symbol": "WETH", "exchangeRate": 2 }
+      ]
+    }
+  }
+}
```

Comment:
I'd probably prefer the config json's to be typescript files. This would also help with recognizing types in the test file. But could be done later.